test/rmath.jl

```diff
@@ -34,6 +34,20 @@ function rmathcomp(basename, params, X::AbstractArray, rtol=_default_rtol(params
     if basename == "srdist"
         has_pdf = false
     end
+    
+    #=
+    R version of signrank has system variation
+    julia> psignrank(18,10,false,true) # windows
+    -0.2076393647782445
+    julia> psignrank(18,10,false,true) # linux
+    -0.20763936477824452
+    This slight difference causes test failures for the inverse functions,
+    due to a slight shift in the location of the discontinuity.
+    =#
+    test_inv = true
+    if basename == "signrank" && Sys.islinux()
+        test_inv = false
+    end
```

Comment:
Maybe simplify this to
```suggestion
    test_inv = basename != "signrank" || !Sys.islinux()
```
?